test/Constraints/overload.swift

```diff
@@ -287,7 +287,22 @@ func rdar97396399() {
 }
 
 // https://github.com/apple/swift/issues/63834
-func f63834(int: Int, string: String) {} // expected-note{{found candidate with type '(Int, String) -> ()'}}
-func f63834(int: Int, string: Bool) {} // expected-note{{found candidate with type '(Int, Bool) -> ()'}}
+func f63834(int: Int, string: String) {} // expected-note 3{{found candidate with type '(Int, String) -> ()'}}
+func f63834(int: Int, string: Bool) {} // expected-note 3{{found candidate with type '(Int, Bool) -> ()'}}
 
-let a = f63834(int:string:) as (Int, Int) -> Void // expected-error{{no exact matches in reference to global function 'f63834'}}
+func f63834_1(int: Int, string: Bool) {} // expected-note{{candidate '(Int) -> Void' requires 1 argument, but 2 were provided}}
+func f63834_1(int: Int, string: String) {} // expected-note{{candidate '(Int) -> Void' requires 1 argument, but 2 were provided}}
+
+func f63834_2(int: Int, string: Bool) {} // expected-note {{found this candidate}}
+func f63834_2(int: Int, string: String) {} // expected-note {{found this candidate}}
```

Comment:
This too. I think we should mention the type of the candidate in the note, as in the first case.